src/main/webapp/components/Table.js

```diff
@@ -156,12 +172,16 @@ export const Table = hh(class Table extends Component {
         striped
         hover
         className='tableContainer'
-        pagination={this.props.reviewFlow}
+        pagination={this.props.pagination}
         search={this.props.reviewFlow}
         options={{
           paginationSize: this.props.paginationSize,
           paginationPosition: 'bottom',
-          sizePerPage: this.props.sizePerPage
+          sizePerPage: this.props.sizePerPage,
+          onSizePerPageList: this.props.onSizePerPageListHandler,
+          onPageChange: this.props.onPageChange,
+          onSearchChange: this.props.onSearchChange,
+          onSortChange: this.props.onSortChange
```

Comment:
Curious ... what happens to other implementations of `Table` that do not provide these props? Does the search page or other components need to do anything specific here?

Reply:
If these callbacks are received as undefined, Bootstrap Table will use their default implementation. So if we do not  pass these props, they will be undefined, and the table will use its default functionality.